src/ui/features/xp-drop/components/XpDropClaimBanner/XpDropClaimBanner.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import DropIcon from 'jsx:src/ui/assets/drop.svg';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { GradientBorder } from 'src/ui/components/GradientBorder';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { Button } from 'src/ui/ui-kit/Button';
+import { Link } from 'react-router-dom';
+
+export function XpDropClaimBanner() {
+  return (
+    <GradientBorder
+      borderColor="linear-gradient(90deg, #a024ef 0%, #fdbb6c 100%)"
```

Comment:
let's move this to var?

Reply:
✅

Reply:
i actually think that inlining may be better:) but ok either way